src/components/checkboxes/_macro.spec.js

```diff
@@ -135,6 +160,15 @@ describe('macro: checkboxes', () => {
         });
     });
 
+    describe('checkbox item with text area', () => {
```

Comment:
We should text the maxlength attribute here to be set to what the char limit should be